kornia/augmentation/_2d/intensity/random_channel_dropout.py

```diff
@@ -0,0 +1,95 @@
+from random import randint, sample
+from typing import Any, Dict, Optional, Tuple, Union
+
+import numpy as np
+from torch import Tensor, clone
+
+from kornia.augmentation._2d.intensity.base import IntensityAugmentationBase2D
+from kornia.core.check import KORNIA_CHECK
+
+
+class RandomChannelDropout(IntensityAugmentationBase2D):
+    """Randomly Drop Channels in the input Image.
+
+    Args:
+        channel_drop_range (int, int): range from which we choose the number of channels to drop.
+        fill_value (int, float): pixel value for the dropped channel.
+        same_on_batch: apply the same transformation across the batch.
+        p: probability of applying the transformation.
+        keepdim: whether to keep the output shape the same as input (True) or broadcast it
+                 to the batch form (False).
+
+    Shape:
+        - Input: :math:`(C, H, W)` or :math:`(B, C, H, W)`
+        - Output: :math:`(B, C, H, W)` or :math:`(C, H, W)`
+
+    Examples:
+        >>> import torch
+        >>> torch.random.initial_seed(1)
+        >>> img = torch.ones(1, 3, 5, 5)
+        >>> out = RandomChannelDropout(p=1)(img)
+        >>> out.shape
+        torch.Size([1, 3, 5, 5])
+        >>> out
+        tensor([[[[False, False, False, False, False],
+          [False, False, False, False, False],
+          [False, False, False, False, False],
+          [False, False, False, False, False],
+          [False, False, False, False, False]],
+
+         [[ True,  True,  True,  True,  True],
+          [ True,  True,  True,  True,  True],
+          [ True,  True,  True,  True,  True],
+          [ True,  True,  True,  True,  True],
+          [ True,  True,  True,  True,  True]],
+
+         [[ True,  True,  True,  True,  True],
+          [ True,  True,  True,  True,  True],
+          [ True,  True,  True,  True,  True],
+          [ True,  True,  True,  True,  True],
+          [ True,  True,  True,  True,  True]]]])
+    """
+
+    def __init__(
+        self,
+        channel_drop_range: Tuple[int, int] = (1, 1),
```

Comment:
```suggestion
        channel_drop_range: tuple[int, int] = (1, 1),
```
`from __future__ import annotations` /cc @johnnv1